driver.go

```diff
@@ -293,6 +353,7 @@ func (d *Driver) buildFingerprint() *drivers.Fingerprint {
 		d.cgroupMgr = info.Host.CgroupManager
 	}
 
+	// TODO: Can we make this more correct with one or the other podman client?
```

Comment:
Yeah for the `driver.podman.version`, `driver.podman.rootless`, and `driver.podman.cgroupVersion`, you'll need to create an attribute for the non-default config and fingerprint each one separately. You can't just interpolate "default" in there, because that'll break existing constraints. See https://github.com/hashicorp/nomad/blob/v1.9.0-beta.1/client/fingerprint/vault.go#L96-L106 for an example of where this kind of thing was added in Nomad for multiple Vault configs.

Reply:
I've added a whole bunch of attributes that are useful to expose, all in the form `driver.podman.<socket name>.<attr>`. This means there's no `driver.podman.version`, `driver.podman.rootless` or `driver.podman.cgroupVersion` anymore. It also doesn't make sense to have a `driver.podman.rootless` since you can combine rootless/root sockets and the value changes per socket.

![Screenshot_2024-09-29_16-26-39](https://github.com/user-attachments/assets/bc56b4a6-e003-4a0a-85f0-2895f78da4b0)

Reply:
There's also a change to the driver struct and how it gives back "healthy" or "unhealthy". Since there's multiple podman sockets, it reports "healthy" if it can connect and retrieve info from all of them. Unhealthy if it cannot connect to some of the sockets and undetected if it cannot connect to any. I've also added a sort on the unresponsive sockets because it'll keep generating logs otherwise: 
![Screenshot_2024-09-29_16-47-52](https://github.com/user-attachments/assets/ed25e992-1916-4857-ade8-607ae1f6fc35)